src/overrides/venia-ui/components/ProductFullDetail/productFullDetail.js

```diff
@@ -222,18 +222,15 @@ const ProductFullDetail = props => {
     const tabsData = [
         {
             tabTitle: "Description",
-            textContentTitle: "title:Product Description",
-            content: productDetails.shortDescription.html ? productDetails.shortDescription.html.replace(/<p>|<\/p>/g, "").trim() : "No short description found"
-            
-            
+            content: [<h1 className={classes.tabTitle}>Product Description</h1>, productDetails.shortDescription.html ? <RichContent html={productDetails.shortDescription.html} /> : <RichContent html={"<p>No short description found</p>"} />]
         },
         {
             tabTitle: "Attributes",
-            textContentTitle: <h1>{'Product Attributes'}</h1>,
-            content: <CustomAttributes customAttributes={customAttributesDetails.list} />
+            content: [<h1 className={classes.tabTitle}>Product Attributes</h1>, <CustomAttributes customAttributes={customAttributesDetails.list} />]
```

Comment:
Instead of passing this as array you can just simply use something like `<React.Fragment>` or `<>...</>` for short.